flutter/news/lib/core/error/failures.dart

```diff
@@ -1,9 +1,14 @@
-import 'package:news/core/result.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+part 'failures.freezed.dart';
 
-class ServerFailure extends Failure {
-  const ServerFailure(String message) : super(message);
+abstract class InternalFailure {}
```

Comment:
I need a name different from `Failure` as that one is now used for one of the two types on the `Result` sealed class. I am open for alternatives